dotnet/src/VectorDataIntegrationTests/PgVectorIntegrationTests/Support/PostgresTestStore.cs

```diff
@@ -18,9 +18,12 @@ internal sealed class PostgresTestStore : TestStore
         .Build();
 
     private NpgsqlDataSource? _dataSource;
+    private string? _connectionString;
 
     public NpgsqlDataSource DataSource => this._dataSource ?? throw new InvalidOperationException("Not initialized");
 
+    public string ConnectionString => this._connectionString ?? throw new InvalidOperationException("Not initialized");
```

Comment:
Some of the tests Dispose the container and I did not want them to dispose the shared `NpgsqlDataSource`, so I had to expose the connection string (with password, as `DataSource.ConnectionString` hides the password)